build.gradle

```diff
@@ -32,7 +32,7 @@ plugins {
     id 'org.jetbrains.dokka' version '1.7.10'
 }
 
-version = '2.6.2-SNAPSHOT'
+version = '3.0.0-beta1-SNAPSHOT'
```

Comment:
Basic question that I am _sure_ it lives somewhere just not sure where, but I assume each release will increment this version?

Reply:
This gets updated on actual releases [here](https://github.com/braintree/browser-switch-android/blob/67c237df2a488d10b1c6c035285a6cbf21501f3e/.github/workflows/release.yml#L86)and [here](https://github.com/braintree/browser-switch-android/blob/main/ci#L40), but not on snapshot releases - so this will remain `3.0.0-beta1-SNAPSHOT` until an actual release on this branch happens (which it won't - we will merge this into main before release)